modules/core/src/main/java/org/locationtech/jts/index/strtree/STRtree.java

```diff
@@ -263,7 +263,30 @@ protected Comparator getComparator() {
     BoundablePair bp = new BoundablePair(this.getRoot(), this.getRoot(), itemDist);
     return nearestNeighbour(bp);
   }
-  
+  /**
+   * Finds k items in this tree which are the top k nearest neighbors to the given {@link Object}, 
+   * using {@link ItemDistance} as the distance metric.
+   * A Branch-and-Bound tree traversal algorithm is used
+   * to provide an efficient search. This method implements the KNN algorithm described in the following paper:
+   * Roussopoulos, Nick, Stephen Kelley, and Frédéric Vincent. "Nearest neighbor queries." ACM sigmod record. Vol. 24. No. 2. ACM, 1995.
+   * <p>
+   * The query <tt>object</tt> does <b>not</b> have to be 
```

Comment:
Have a look at http://stackoverflow.com/questions/1667212/how-to-add-reference-to-a-method-parameter-in-javadoc

It should probably say {@code item}, or else maybe {@ link Object} (although I think the former is more specific)